utils/modular_model_converter.py

```diff
@@ -632,173 +894,144 @@ def replace_call_to_super(
     "Processor": "processing",
     "ImageProcessor": "image_processing",
     "FeatureExtractor": "feature_extractor",
+    "ProcessorKwargs": "processing",
+    "ImagesKwargs": "processing",
+    "TextKwargs": "processing",
```

Comment:
Are we expecting clashes in the names here? Because infering based on the class using them is not so straightforward since different classes in different file types may use them (e.g. type hints)

Reply:
For what I can see, all of them are only used in "processing" files as of now at least